ts/src/hyperliquid.ts

```diff
@@ -874,7 +874,12 @@ export default class hyperliquid extends Exchange {
         //         }
         //     ]
         //
-        return this.parseOHLCVs (response, market, timeframe, since, limit);
+        const results = [];
+        for (let i = 0; i < response.length; i++) {
+            results.push (this.parseOHLCV (response[i], market));
+        }
+        const sorted = this.sortBy (results, 0);
```

Comment:
we do this inside `parseOHLCVs` no?

Reply:
ok the problem is not the order but the `tail` cut

Reply:
we should probably update parseOHLCVs to fix this, I will do it on a different PR, can you revert these changes?